release/connect-streams.js

```diff
@@ -32409,12 +32436,6 @@ AWS.apiLoader.services['connect']['2017-02-15'] = require('../apis/connect-2017-
       });
       this.agent.configuration.routingProfile.routingProfileId =
         this.agent.configuration.routingProfile.routingProfileARN;
-      
-      if (this.suppress) {
```

Comment:
Removing this will break DR Streams

Reply:
I don't think we push DR specific code publicly yet.

Reply:
spoke offline, NVM this was my bad